tests/disaster-recovery/sc_arbiter/test_add_capacity.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import logging
+
+from datetime import datetime, timezone
+from ocs_ci.framework.pytest_customization.marks import (
+    turquoise_squad,
+    stretchcluster_required,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import storage_cluster
+from ocs_ci.ocs.resources.pod import (
+    get_pod_restarts_count,
+    get_ceph_tools_pod,
+    wait_for_pods_to_be_in_statuses,
+)
+from ocs_ci.ocs.resources.stretchcluster import StretchCluster
+
+logger = logging.getLogger(__name__)
+
+
+@turquoise_squad
+@stretchcluster_required
+class TestAddCapacityStretchCluster:
+    """
+    Add capacity to the Stretch cluster with arbiter configuration
+
+    """
+
+    @staticmethod
+    def add_capacity_to_stretch_cluster():
+        """
+        Perform add capacity on a stretch cluster
+
+        """
+        # get osd pods restart count before
+        osd_pods_restart_count_before = get_pod_restarts_count(
+            label=constants.OSD_APP_LABEL
+        )
+
+        # add capacity to the cluster
+        storage_cluster.add_capacity_lso(ui_flag=False)
+        logger.info("Successfully added capacity")
+
+        # get osd pods restart count after
+        osd_pods_restart_count_after = get_pod_restarts_count(
+            label=constants.OSD_APP_LABEL
+        )
+
+        # assert if any osd pods restart
+        assert sum(osd_pods_restart_count_before.values()) == sum(
+            osd_pods_restart_count_after.values()
+        ), "Some of the osd pods have restarted during the add capacity"
+        logger.info("osd pod restarts counts are same before and after.")
+
+        # assert if osd weights for both the zones are not balanced
+        tools_pod = get_ceph_tools_pod()
+        zone1_osd_weight = tools_pod.exec_sh_cmd_on_pod(
+            command=f"ceph osd tree | grep 'zone {constants.DATA_ZONE_LABELS[0]}' | awk '{{print $2}}'",
+        )
+        zone2_osd_weight = tools_pod.exec_sh_cmd_on_pod(
+            command=f"ceph osd tree | grep 'zone {constants.DATA_ZONE_LABELS[1]}' | awk '{{print $2}}'",
+        )
+
+        assert float(zone1_osd_weight.strip()) == float(
+            zone2_osd_weight.strip()
+        ), "OSD weights are not balanced"
+        logger.info("OSD weights are balanced")
+
+    @pytest.mark.last
+    @pytest.mark.parametrize(
+        argnames=["iterations"],
+        argvalues=[
+            pytest.param(
+                3,
+                marks=[
+                    pytest.mark.polarion_id("OCS-5474"),
+                    pytest.mark.bugzilla("2143858"),
+                ],
+            ),
+        ],
+    )
```

Comment:
not a big deal - is there a reason to use `@pytest.mark.parametrize`, given there is a single test iteration here?

Reply:
@ebenahar yeah the reason behind this is to avoid modifying the core test function code if any changes to interations needed